InferenceSystem/requirements.txt

```
@@ -25,4 +25,4 @@ opencv-python
 boto3
 pytz
 opencensus-ext-azure
-orca-hls-utils
+git+https://github.com/dthaler/orca-hls-utils.git@master#egg=orca-hls-utils
```

```suggestion
git+https://github.com/orcasound/orca-hls-utils.git@master#egg=orca-hls-utils
```
Switch back to upstream orca-hls-utils